cli/tests/unit/read_text_file_test.ts

```diff
@@ -145,3 +145,24 @@ Deno.test(
     assert(data.length > 0);
   },
 );
+
+Deno.test(
+  { permissions: { read: true, write: true } },
+  function throwErrorOnReadBigTextFileSync() {
+    const kStringMaxLengthPlusOne = 536870888 + 1;
+    const bytes = new Uint8Array(kStringMaxLengthPlusOne);
+    const filePath = "cli/tests/testdata/too_big_a_file.txt";
+
+    Deno.writeFileSync(filePath, bytes);
```

Comment:
How long does it take to run this test?

Reply:
`throwErrorOnReadBigTextFileSync ... ok (935ms)`